politeiawww/cmsdatabase/cockroachdb/cockroachdb.go

```diff
@@ -376,6 +377,91 @@ func (c *cockroachdb) InvoicesByDateRangeStatus(start, end int64, status int) ([
 	return dbInvoices, nil
 }
 
+// InvoicesByLineItemsProposalToken takes a proposal token as an argument and
+// returns all invoices that have line items corresponding with that token.
+// All line items that are not considered relevant to the proposal token will
+// be omitted.
+func (c *cockroachdb) InvoicesByLineItemsProposalToken(token string) ([]*database.Invoice, error) {
+	log.Debugf("InvoicesByLineItemsProposalToken: %v", token)
+	// Get all line items with proposal token
+	query := `SELECT 
```

Comment:
Can you replace the tabs with spaces in these string literals so that the indentation lines up?